docs/index.rst

```diff
@@ -61,6 +61,75 @@ Quick Start
 
    .. note:: It is always recommended to use python virtual environment
 
+.. _how_it_works:
```

Comment:
Not sure but lines like this one may be breaking the left menu on the docs see the "Usage Examples" and the items below it on the current documentation.